lakefile.lean

```diff
@@ -26,25 +26,7 @@ require "leanprover-community" / "plausible" @ git "main"
 /-- These options are used as `leanOptions`, prefixed by `` `weak``, so that
 `lake build` uses them, as well as `Archive` and `Counterexamples`. -/
 abbrev mathlibOnlyLinters : Array LeanOption := #[
```

Comment:
(Without testing) I think this is equivalent to the previous setup:
```suggestion
abbrev mathlibOnlyLinters : Array LeanOption := #[
  ⟨`linter.mathlibStandardSet, true⟩,
  ⟨`linter.style.longFile, .ofNat 1500⟩,
```

Reply:
Oh, I forget about that linter :see_no_evil: So you cannot set non-boolean options in linter sets, right?
(Is that a useful feature?)

Reply:
I'm not sure about the semantics of adding non-boolean options to sets, especially when multiple sets are enabled: which value do we choose? So I think we can leave this a separate option for now. We could also split this linter into a boolean enable/disable and a numeric max length.

Reply:
I think leaving this as-is for now makes sense. (Also, from the perspective of downstream projects, the file length is certainly not the most important linter to enable.)